src/main/java/com/integerlimit/deprecatedapi/api/DeprecatedItem.java

```diff
@@ -1,11 +1,15 @@
 package com.integerlimit.deprecatedapi.api;
 
+import net.minecraft.item.ItemStack;
+
 /**
  * A deprecated item. Has a tooltip message.
  */
 public class DeprecatedItem {
     public static final String defaultMessage = "Deprecated";
     private String tooltipMessage;
+    protected ItemStack stack = ItemStack.EMPTY;
```

Comment:
If we are storing an ItemStack, then you can just store the itemstack, and not store meta, as itemstack contains data of meta

Aka, use `new ItemStack(Item item, int metadata, int count)`

Just put in a base value (1) for count

Reply:
Ah. Can we store Item instead?